packages/core/package.json

```diff
@@ -24,10 +42,74 @@
     "chalk": "^5.0.0"
   },
   "dependencies": {
-    "@bluecadet/launchpad-utils": "*",
-    "@bluecadet/launchpad-dashboard": "*",
-    "@bluecadet/launchpad-monitor": "*",
-    "@bluecadet/launchpad-scaffold": "*",
-    "@bluecadet/launchpad-content": "*"
-  }
+    "@bluecadet/launchpad-utils": "~1.0.1",
+    "@bluecadet/launchpad-dashboard": "~1.0.1",
+    "@bluecadet/launchpad-monitor": "~1.0.1",
+    "@bluecadet/launchpad-scaffold": "~1.0.1",
+    "@bluecadet/launchpad-content": "~1.0.1",
+    "@portabletext/to-html": "^1.0.3",
+    "@sanity/block-content-to-markdown": "^0.0.5",
+    "@sanity/client": "^3.3.2",
+    "@sindresorhus/slugify": "^2.1.0",
+    "airtable": "^0.11.1",
+    "async": "^3.2.3",
+    "auto-bind": "^5.0.1",
+    "axon": "^2.0.3",
+    "chalk": "^5.0.0",
+    "cli-progress": "^3.9.1",
+    "contentful": "^9.0.0",
+    "cross-spawn": "^7.0.3",
+    "formdata-node": "^4.3.1",
+    "fs-extra": "^10.0.0",
+    "get-urls": "^10.0.0",
+    "got": "^11.8.2",
+    "jsonpath": "^1.1.1",
+    "logform": "^2.3.2",
+    "markdown-it": "^12.2.0",
+    "moment": "^2.29.1",
+    "p-debounce": "^4.0.0",
+    "p-queue": "^7.1.0",
+    "pm2": "^5.1.2",
+    "rimraf": "^3.0.2",
+    "sanitize-html": "^2.5.1",
+    "semver": "^7.3.5",
+    "sharp": "^0.29.1",
+    "strip-json-comments": "^4.0.0",
+    "sudo-prompt": "^9.2.1",
+    "tail": "^2.2.4",
+    "winston": "^3.4.0",
+    "winston-daily-rotate-file": "^4.5.5",
+    "yargs": "^17.3.1"
```

Comment:
A small housekeeping task that might make sense to do in this PR is move a lot of these dependencies into the relevant `package.json`s, rather than having them as peer dependencies. 

Right now, if someone were to try to install only `@launchpad/content`, they wouldn't be able to run it without also installing all of the `@launchpad/content` package's peer-dependencies.

Reply:
Totally! TBH pulling up these dependencies and using the `file://` syntax were both results of NPM bugs back when I put this together. Glad you found a solution that works now!

Reply:
Ah gotcha, that makes sense.

Reply:
fixed in commit 3461b84